natives.json

```diff
@@ -3778,16 +3778,16 @@
 			"build": "1207"
 		},
 		"0x0F2A2175734926D8": {
-			"name": "_0x0F2A2175734926D8",
+			"name": "_STOP_SOUND",
```

Comment:
This name can be a little confusing since one with the same name already exists in GTAV (takes a sound ID instead though). I'd suggest to name it something like ``_STOP_SOUND_WITH_NAME`` to make sure there will be no confusion (especially if R* adds back STOP_SOUND).